index.html

```diff
@@ -11,250 +11,339 @@
               A daily selection of privately personalized reads; no accounts or
-              sign-ups required. has been the industry's standard
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
             </p>
-            <ul class="card-tool">
-              <li class="tools tools-spec">html</li>
-
-              <li class="tools tools-spec bootstrap">bootstrap</li>
-
-              <li class="tools tools-spec">Ruby</li>
+            <ul class="technology">
+              <li class="tools">css</li>
+              <li class="tools">html</li>
+              <li class="tools bootstrap">bootstrap</li>
+              <li class="tools">Ruby</li>
             </ul>
-            <button type="button" class="button btn">see project</button>
+            <button type="button" class="button">see project</button>
           </div>
         </div>
-      </article>
-      <article>
-        <div class="cards">
-          <div class="card-content">
-            <h2 class="cd-header">profesional art printing data</h2>
-            <p class="cd-txt">
-              A daily selection of privately personalized reads; no accounts or
-              sign-ups required. has been the industry's standard
-            </p>
-            <ul class="card-tool">
-              <li class="tools tools-spec">html</li>
+        <div class="grid-container">
+          <article>
+            <div class="cards cd-1">
+              <div class="card-content hide">
+                <h2 class="cd-header">profesional art printing data</h2>
+                <p class="cd-txt">
+                  A daily selection of privately personalized reads; no accounts
+                  or sign-ups required. has been the industry's standard
+                </p>
+                <ul class="card-tool">
+                  <li class="tools tools-spec">html</li>
 
-              <li class="tools tools-spec bootstrap">bootstrap</li>
+                  <li class="tools tools-spec bootstrap">bootstrap</li>
 
-              <li class="tools tools-spec">Ruby</li>
-            </ul>
-            <button type="button" class="button btn">see project</button>
-          </div>
-        </div>
-      </article>
-      <article>
-        <div class="cards">
-          <div class="card-content">
-            <h2 class="cd-header">profesional art printing data</h2>
-            <p class="cd-txt">
-              A daily selection of privately personalized reads; no accounts or
-              sign-ups required. has been the industry's standard
-            </p>
-            <ul class="card-tool">
-              <li class="tools tools-spec">html</li>
+                  <li class="tools tools-spec">Ruby</li>
+                </ul>
+              </div>
+              <button type="button" class="button btn">see project</button>
+            </div>
```

Comment:
- Good job designing the project cards but it lacks some functionality.
- [ ] The description of each of these project cards should be on top of their image with a dark opacity and with a white color text and when the project is hovered on with the mouse the description and dark opacity disappear and a `See Project` button appears to the bottom of the image. It would be great if you could implement this functionality in all your project cards.
- [ ] Also your card elements start leaving the card as the screen size increases. Kindly address this issue 👍 
- Your design:
![Annotation 2022-02-26 065637](https://user-images.githubusercontent.com/85113567/155831459-a5f66951-3b62-40f0-8732-16fef8341cae.jpg)
- Figma design: 
![Annotation 2022-02-26 064552](https://user-images.githubusercontent.com/85113567/155831469-3abb406a-b17e-4f02-880b-b699c846cd34.jpg)

Reply:
well resolved 💯